### DISCUSSION TITLE
Add a scale to rendered scene

### DISCUSSION THREAD

**Discussion by dominformant** [NVlabs/sionna#712] [Q&A] [answered]
Hi everyone, first thanks for developing sionna. I'm wondering, whether there is a simple option to add a scale to a rendered scene with a coverage map in order to visualize the dimensions of the scene. I know, that the coverage_map.show() method provides axis, however I would prefer using a rendered scene as this also shows the simulated buildings.

**Comment by merlinND** [comment, marked as answer]
Hello @dominformant,

Since each user may use different scene units, as well as choose the camera viewpoint arbitrarily, it would be easier for the user to annotate the Matplotlib figure returned by `scene.render()` themselves.

**Reply by dominformant** [reply]
Thanks for your reply, that's actually what I'm currently doing. I was just wondering whether there is an option to do this "automatically" I did not find yet.
